Sources/StreamChat/Controllers/MessageController/MessageController.swift

```diff
@@ -243,20 +243,22 @@ public class ChatMessageController: DataController, DelegateCallable, DataStoreP
     ///
     /// - Parameters:
     ///   - text: The updated message text.
+    ///   - skipEnrichUrl: If true, skips url enriching.
     ///   - attachments: An array of the attachments for the message.
     ///   - extraData: Custom extra data. When `nil` is passed the message custom fields stay the same. Equals `nil` by default.
     ///   - completion: The completion. Will be called on a **callbackQueue** when the network request is finished.
     ///                 If request fails, the completion will be called with an error.
-    ///
     public func editMessage(
         text: String,
+        skipEnrichUrl: Bool = false,
```

Comment:
About the second part - we should try to follow the REST API as closely as possible. It helps with consistency with the dashboard and other SDKs. 
In general, I think we don't even need a param here, it should all be controlled from the config on the dashboard.

Reply:
I agree @laevandus, but we already have other params in the function, so it would be weird to have some "options" outside of the options structure. And since we can't do breaking changes, I think in this case it makes sense to be consistant and not introduce an `options` structure, especially because `createNewMessage` also has multiple params instead of the `options`approach. I also prefer the `options` struct, and we are using it already in newer APIs. But for this one, to keep consistent we probably should stick with regular params 👍